rest-api-spec/src/main/resources/rest-api-spec/api/exists.json

```diff
@@ -42,6 +46,27 @@
         "routing": {
           "type" : "string",
           "description" : "Specific routing value"
+        },
+        "_source": {
+          "type" : "list",
+          "description" : "True or false to return the _source field or not, or a list of fields to return"
+        },
+        "_source_exclude": {
+          "type" : "list",
+          "description" : "A list of fields to exclude from the returned _source field"
+        },
+        "_source_include": {
+          "type" : "list",
+          "description" : "A list of fields to extract and return from the _source field"
+        },
+        "version" : {
+          "type" : "number",
+          "description" : "Explicit version number for concurrency control"
```

Comment:
@tlrx I pushed 0c63bf3ef687813524a88bb4fcb1b676e3dd83d2. This test would have failed before #23186, and passes after it.